vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.h

```diff
@@ -16,6 +16,19 @@ struct VirtualTextShadowNode final : public ShadowNodeBase {
   TransformableText transformableText{};
 
   void AddView(ShadowNode &child, int64_t index) override;
+
+  bool m_isHighlighted = false;
+  folly::dynamic m_ColorValue = nullptr;
+  size_t m_length = 0;
+
+  typedef struct HighlightData {
+    std::vector<std::unique_ptr<HighlightData>> data;
+    size_t spanIdx = 0;
+    bool isHighlighted = false;
+    folly::dynamic color;
+  } HighlightData;
```

Comment:
we're trying to limit the number of places where we use folly so we can remove it at some point, can we use winrt::Color instead (or even better just a DWORD for argb)

Reply:
Ok, I'll convert from folly::dynamic to winrt::Color and store that.